ambry-tools/src/main/java/com.github.ambry/store/DumpIndexTool.java

```diff
@@ -64,19 +65,25 @@
   private final boolean activeBlobsOnly;
   // True if blob stats needs to be logged, false otherwise
   private final boolean logBlobStats;
+  // set to true if only error logging is required
+  private final boolean silent;
```

Comment:
setting log levels in lgo4j, will enable in all packages. Also even in this package, there might be extraneous statements debug statements which may not be intended. Basically we want to differentiate between 3 intentions. 1: Dumping index in itself (which would required just the index entries to be dumped. 2: ConsistencyCheckerTool where in only errors might be required. 3: Extra trace statements for debugging purposes. Hence, have introduced the "silent" option and have removed the "logBlobStats" to avoid confusion.